src/kleinanzeigen_bot/model/ad_model.py

```
@@ -4,13 +4,15 @@
 from __future__ import annotations
 
 import hashlib, json  # isort: skip
+from collections.abc import Mapping, Sequence
 from datetime import datetime  # noqa: TC003 Move import into a type-checking block
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `TC003`)

Remove unused `noqa` directive

(RUF100)